cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -283,7 +283,9 @@ export class CanvasViewImpl implements CanvasView, Listener {
             }
         }
 
-        if (data) {
+        const emptyMask = data && data.shapeType === 'mask' && data.points.length < 6;
```

Comment:
```suggestion
        const isEmptyMask = data && data.shapeType === 'mask' && data.points.length < 6;
```

Reply:
I believe would be better to check a mask is not empty for both editing and drawing in `masksHandler.ts`.

Reply:
Do you mean calling `onEditDone(null)`/`onDrawDone(null)` in `draw`/`edit`?

Reply:
Yep